syncd/Syncd.cpp

```diff
@@ -920,6 +1097,35 @@ sai_status_t Syncd::processBulkOid(
 
     sai_status_t all = SAI_STATUS_SUCCESS;
 
+#ifdef SAI_SUPPORT_BULK_API
```

Comment:
as mentioned before, make this command line flag, and also  split processBulkOid to 2 methods 1st to use bulk vendor API and 2nd that will use unwrapped loop using single vendor api